src/Text/Layout/Table/LineStyle.hs

```diff
@@ -0,0 +1,167 @@
+module Text.Layout.Table.LineStyle where
+
+import Data.Function (on)
+
+-- | The line styles supported by the Unicode Box-Drawing block.
+data LineStyle
+    = NoLine
+    | SingleLine
+    | HeavyLine
+    | DoubleLine
+    | DashLine
+    | HeavyDashLine
+    | Dash4Line
+    | HeavyDash4Line
+    | Dash2Line
+    | HeavyDash2Line
```

Comment:
Can you explain what these mean?  This is not self-explanatory.

Reply:
NoLine: ` `/` `
SingleLine: `─`/`│`
HeavyLine: `━`/`┃`
DoubleLine: `═`/`║`
DashLine: `┄`/`┆`
HeavyDashLine: `┅`/`┇`
Dash4Line: `┈`/`┊`
HeavyDash4Line: `┉`/`┋`
Dash2Line: `╌`/`╎`
HeavyDash2Line: `╍`/`╏`

Reply:
Please document then. For example:
```
    -- | @─@  and @│@.
    SingleLine 
```